src/integTest/groovy/net/wasdev/wlp/gradle/plugins/TestSpringBootApplication.groovy

```diff
@@ -134,5 +134,44 @@ public class TestSpringBootApplication extends AbstractIntegrationTest{
         }
     }
 
+    //@Test
+    public void test_spring_boot_apps_with_classifier_15() {
+        try {
+            /*
+            addApplToServerXml("src/main/liberty/config/server.xml",
```

Comment:
Might be able to remove these xml editing methods if you use different server.xml files with different build.gradle files for each of these test builds.

Reply:
removed